### DISCUSSION TITLE
How do I get the Heart On Your Sleeve Badge?

### DISCUSSION THREAD

**Discussion by samfreitasxs** [community/community#43514] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

![image](https://user-images.githubusercontent.com/37847692/211087395-acb7cf8a-f595-4345-955b-f99937c833ca.png)

How do I get the Heart On Your Sleeve Badge?

**Comment by Igorcbraz** [comment]
possibly it was a badge for an event or something, I don't know if we have any way of knowing

**Reply by techguruCD** [reply]
❤

**Reply by DanielTsekhanovskyi** [reply]
❤

**Reply by LorisYanis** [reply]
❤️

**Reply by BLazzeD21** [reply]
❤️

**Reply by JoeyKhd** [reply]
❤️

**Reply by sh0rtener** [reply]
is it fun to put ❤️ everywhere?

**Reply by MrBElga** [reply]
❤️

**Reply by Harshkhosla** [reply]
❤️

**Reply by JhonnSantiagoBernalJuradoCampus** [reply]
❤️

**Reply by QuentinRiviere** [reply]
❤️

**Reply by a5512167086** [reply]
❤️

**Reply by asareynolds** [reply]
❤️

**Reply by fazrilrama** [reply]
❤️

**Reply by LeToyek** [reply]
❤️

**Reply by bilalmirza74** [reply]
❤️

**Reply by ManChrys** [reply]
❤️

**Reply by GsnMithra** [reply]
❤️

**Reply by fazrilrama** [reply]
❤️

**Reply by dev-ugurkontel** [reply]
❤️

**Reply by dev-ugurkontel** [reply]
👋

**Reply by dev-ugurkontel** [reply]
👋

**Reply by dev-ugurkontel** [reply]
👋🏻

**Reply by dev-ugurkontel** [reply]
👋🏼

**Reply by dev-ugurkontel** [reply]
👋🏽

**Reply by dev-ugurkontel** [reply]
👋🏾

**Reply by dev-ugurkontel** [reply]
👋🏿

**Reply by fazrilrama** [reply]
👋

**Reply by Ramadhann** [reply]
👋

**Reply by fazrilrama** [reply]
👋

**Reply by Ramadhann** [reply]
👋

**Reply by sambabhouria** [reply]
👋

**Reply by themagicalmammal** [reply]
❤️

**Reply by IllyaKh** [reply]
❤️

**Reply by mkdirer** [reply]
❤️

**Reply by lucalm2004** [reply]
❤️

**Reply by jungwoo3490** [reply]
❤️

**Reply by MacTii** [reply]
❤

**Reply by StepJes** [reply]
❤

**Reply by pedrohma07** [reply]
❤

**Reply by AndKlet** [reply]
❤️

**Reply by yooseungmo** [reply]
❤

**Reply by backtoaldo** [reply]
❤️

**Reply by EdamAme-x** [reply]
❤

**Reply by wei4102706** [reply]
❤

**Reply by LazyFossil** [reply]
❤

**Reply by YashvPatel** [reply]
❤

**Reply by s1lentf00thold** [reply]
❤️

**Reply by Estebanbss** [reply]
❤️

**Reply by Estebanbss** [reply]
❤️

**Reply by Git-SalahEdd** [reply]
❤

**Reply by muhammedkurtulus** [reply]
❤️

**Reply by pg56714** [reply]
❤️

**Reply by nick-jy-huang** [reply]
❤️

**Reply by bartzalewski** [reply]
❤️

**Reply by hayden-donnelly** [reply]
❤️

**Reply by yang-aaron** [reply]
❤️

**Reply by odilson-dev** [reply]
❤

**Reply by om04an** [reply]
Спасибо ✌️

**Reply by ghost** [reply]
❤️

**Reply by gem-harshil** [reply]
❤️

**Reply by pecraveiro** [reply]
❤️

**Reply by kauaneiras** [reply]
I don't think it's possible to earn this achievement yet, because I haven't found any repository with anyone who has won it.

**Reply by chzuhadulhadi** [reply]
❤

**Reply by slepimis120** [reply]
❤️

**Reply by kanekoshoyu** [reply]
:heart:

**Reply by Jhonatha1** [reply]
<3

**Reply by Oscar-M-9** [reply]
❤️

**Reply by shakeerk123** [reply]
❤️

**Reply by andriimazurets** [reply]
❤️

**Reply by ardana96** [reply]
❤️

**Reply by kirubel606** [reply]
🤣😁👏🏽👏🏽

**Reply by OsemaFadhel** [reply]
❤️

**Reply by Abhay2201-Av** [reply]
❤️

**Reply by bumblebee-sh** [reply]
❤️

**Reply by thuansan** [reply]
❤️

**Reply by 14790897** [reply]
❤️

**Reply by Weile-Zheng** [reply]
❤️

**Reply by dzc0d3r** [reply]
❤️

**Reply by ZuhairHossain** [reply]
❤️

**Reply by datntse150392** [reply]
❤️

**Reply by onestn** [reply]
❤️

**Reply by EdisonGualle** [reply]
❤️

**Reply by watan104** [reply]
❤️

**Reply by MuchaSsak** [reply]
❤️

**Reply by sante-afk** [reply]
❤️

**Reply by Semigradsky** [reply]
:heart:

**Reply by Kabilduke** [reply]
❤️

**Reply by julijacibulska** [reply]
❤️

**Reply by henrik-klev** [reply]
❤️

**Reply by egebilecen** [reply]
❤️

**Reply by OskarWiedeweg** [reply]
❤️

**Reply by koliy82** [reply]
❤️

**Reply by shenjingnan** [reply]
❤️

**Reply by ZubkoKarina** [reply]
❤️

**Reply by debdeep-ghosh** [reply]
❤️

**Reply by tneils218** [reply]
❤️

**Reply by shutylova** [reply]
❤️

**Reply by yogawa** [reply]
❤️

**Reply by Mugens** [reply]
❤️

**Reply by mozi47** [reply]
❤️

**Reply by makstron** [reply]
❤️

**Reply by Martin-Aziz** [reply]
❤️

**Reply by um-xair** [reply]
❤️

**Reply by HoussemDAAS** [reply]
❤️

**Reply by atilagulers** [reply]
❤️

**Reply by erenozy66** [reply]
❤️

**Reply by daguerpedro** [reply]
❤

**Reply by Husain-Jinia** [reply]
❤

**Reply by MattiaValerio** [reply]
❤️

**Reply by alexandroivaldez** [reply]
❤

**Reply by ChillBroh** [reply]
❤

**Reply by CoreTiQ** [reply]
❤

**Reply by vas11111** [reply]
❤

**Reply by HHDVasishtPranav** [reply]
❤

**Reply by iamRahulKumarr** [reply]
❤️

**Reply by NikharVerma** [reply]
❤️

**Reply by joseminelli** [reply]
❤️

**Reply by madmarin** [reply]
❤️

**Reply by EkoEdyP** [reply]
❤️

**Reply by lzz19980125** [reply]
❤️

**Reply by callmenoway** [reply]
❤️

**Reply by Randy0607** [reply]
❤️

**Reply by bstr1ckland** [reply]
❤️

**Comment by melvincwng** [comment, marked as answer]
Hey @samfreitasxs, the 'Heart On Your Sleeve' & 'Open Sourcerer' achievement badges are not yet released to public GitHub users at the moment.

Please kindly wait for & stay tuned to the official news release by GitHub on their blog over [here](https://github.blog/). 

When they launched the first set of achievement badges, they previously also posted a blog over [here](https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/) to let users know more information. Hence, they might do the same thing (i.e. post a new blog) to announce updates to the achievement system on GitHub.

Other than that, there has been no information (based on my understanding) on the mechanics to achieve those two new badges.

Hope this clarifies & answers your question, take care Samuel 😄!

**Reply by QuocHuy03** [reply]
hello

**Reply by s1lentf00thold** [reply]
❤️

**Reply by j-refs** [reply]
Ok, understandable, but I must say, I am completely outraged that GitHub has barely given anything to Linus Torvalds!



 Like, he literally made the only good operating system available! AND HE MADE IT FREE!!!!

Like, idk, if IIIII had a whole successful company literally built on the back on someone's ground breaking, I'd be kissing a lot of ass... Just saying.

Like I cant believe GitHub gave Linus himself the most basic badges ever!!

**Reply by farid-teymouri** [reply]
❤️

**Reply by JesusOsama** [reply]
❤️

**Reply by mkos11** [reply]
Hey @samfreitasxs, thanks for bringing this to my attention! It seems like the 'Heart On Your Sleeve' & 'Open Sourcerer' achievement badges are not yet available for public GitHub users. The best way to stay updated on these releases is to keep an eye on GitHub's official blog. They usually announce updates and new features there.

It's possible that GitHub will share more details, including the mechanics to achieve these badges, in a blog post just like they did with the first set of achievement badges.

For now, let's stay tuned for the official news release from GitHub. I'll be sure to keep you posted if there are any updates. Feel free to reach out if you have any more questions or if there's anything else I can help you with. Take care and happy coding! 😄🚀

**Reply by umartynau** [reply]
❤️

**Reply by datntse150392** [reply]
❤️

**Reply by gritlm** [reply]
❤️

**Reply by deykun04** [reply]
❤️

**Reply by nhanvnguyen** [reply]
<3 <3

**Reply by mohmmadView** [reply]
❤️

**Reply by sante-afk** [reply]
❤️

**Reply by AjayManikantaG** [reply]
❤️

**Reply by Semigradsky** [reply]
:heart:

**Reply by AAndrei06** [reply]
❤️

**Reply by jenny83318** [reply]
❤️

**Reply by Otavio15** [reply]
:heart:

**Reply by Bergan404** [reply]
❤️

**Reply by devmuhib009** [reply]
Thanks for the clarification. I was trying it. After reading your comment, I can move forward.

**Reply by JahongirHakimjonov** [reply]
❤️

**Reply by sunnyshin8** [reply]
❤️

**Reply by tneils218** [reply]
❤️

**Reply by hamzehazizzadeh** [reply]
❤️

**Reply by ElianBorda** [reply]
❤

**Reply by ainzone** [reply]
❤

**Reply by phuccoder2000** [reply]
❤

**Reply by Navya2ak** [reply]
❤️

**Reply by CoreTiQ** [reply]
❤

**Reply by Andresmup** [reply]
❤️

**Reply by DMurawiecki** [reply]
❤️

**Reply by willyg23** [reply]
❤️

**Reply by madmarin** [reply]
❤️

**Reply by CC11001100** [reply]
❤️

**Reply by EkoEdyP** [reply]
❤️

**Reply by hust-whw** [reply]
❤

**Reply by KarolannMauger** [reply]
❤

**Comment by Tynab** [comment]
To earn the Heart on Your Sleeve achievement on GitHub, you need to publicly share a project on GitHub that contains a valid license and a README file. Here are the steps you can follow:
1. Sign up for a GitHub account if you haven't already.
2. Create a new project or select an existing project that you want to share on GitHub. The project should include a valid open source license and a README file that provides information about the project.
3. Create a new repository on GitHub and upload your project files.
4. Add a description and any other relevant information about your project, such as installation instructions or usage examples, to the README file.
5. Choose an appropriate open source license for your project and include a LICENSE file in the repository. GitHub provides several options for open source licenses that you can choose from.
6. Make sure that the repository is set to "public" so that others can view and access your project.
7. After you have completed these steps, you should receive the Heart on Your Sleeve badge on your GitHub profile.
Note that GitHub has specific requirements for what constitutes a valid open source license, so be sure to review the license options and choose one that fits the needs of your project. Additionally, it's important to provide clear and comprehensive information about your project in the README file to help others understand how to use and contribute to your project. Good luck!

**Reply by seldish-og** [reply]
why didn't u earn it yorself?

**Reply by JavierRangel18** [reply]
👀

**Reply by eslamalbaik** [reply]
You are lying, man

**Reply by Young-Lord** [reply]
ChatGPT is so useful LOL

**Reply by fazrilrama** [reply]
👀

**Reply by Buddhikanip** [reply]
I don't think it works, GitHub docs don't say about this.

**Reply by mkos11** [reply]
Great use chatgpt     ...................................

**Reply by RamirJunior** [reply]
I'd like to get this badge, can someone put the Heart in one of my projects? Thank you so much!
But I don't think that it works XD

**Comment by ShivgunGaming** [comment]
Is Heart of the Sleeve obtainable now @melvincwng

**Comment by montasim** [comment]
Hi @everyone, I am trying to get the "Heart On Your Sleeve" badge. How do I get it?

**Reply by willyg23** [reply]
❤️

**Comment by eslamalbaik** [comment]
It is not visible to anyone, only a GitHub employee can ring it

**Comment by JustAlexeyDev** [comment]
Hello there! Can I get it?

**Comment by MadhushaPrasad** [comment]
Hello there! You can get more details about these badge by referring this URL : [here](https://github.com/Schweinepriester/github-profile-achievements).


And you have to stay tuned to the official news release by GitHub on their blog over [here](https://github.blog/).

**Reply by fazrilrama** [reply]
👀

**Reply by JhonnSantiagoBernalJuradoCampus** [reply]
Omg

**Comment by Fodase4** [comment]
Em português.  O que tenho que fazer?

**Comment by Maham-j** [comment]
Hello there! Can I get hand-on sleeves badge?

**Reply by Maham-j** [reply]
mark my answer as accepted so i get galaxy brain badge

**Comment by KalioNix** [comment]
Hello there!

**Comment by ShihabHashib** [comment]
Is it still possible to get "Heart On Your Sleeve"?

**Comment by Nirzak** [comment]
It’s still not released yet. So criteria is unknown. A rumoured criteria is React to something on GitHub with a ❤️ emoji

**Reply by iprattham** [reply]
yes

**Comment by s1lentf00thold** [comment]
Its Easy To get

**Comment by iprattham** [comment]
Hey @samfreitasxs , the 'Heart On Your Sleeve' & 'Open Sourcerer' achievement badges are not yet released to public GitHub users at the moment.
As it is under the beta version now and is under going tests whether it would work fine or not.
Kindly wait for & stay tuned to the official news release by GitHub on their blog over [here](https://github.blog/).

When they launched the first set of achievement badges, they previously also posted a blog over [here](https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/) to let users know more information. Hence, they might do the same thing (i.e. post a new blog) to announce updates to the achievement system on GitHub.

Other than that, there has been no information (based on my understanding) on the mechanics to achieve those two new badges.

Hope this clarifies & answers your question, have a great day!

**Reply by dodolereporter** [reply]
❤️

**Comment by vedant-achh** [comment]
❤️

**Comment by themagicalmammal** [comment]
❤️

**Comment by kauaneiras** [comment]
I don't think it's possible to earn this achievement yet, because I haven't found any repository with anyone who has won it.

**Comment by aseralwrd715** [comment]
اىجو المساعده لقد اخترق حسابي

---------- Forwarded message ---------
من: Kauan T. Eiras ***@***.***>
‪Date: الثلاثاء، ٧ نوفمبر ٢٠٢٣, ٢:٢٣ ص‬
Subject: Re: [community/community] How do I get the Heart On Your Sleeve
Badge? (Discussion #43514)
To: community/community ***@***.***>
Cc: Subscribed ***@***.***>


I don't think it's possible to earn this achievement yet, because I haven't
found any repository with anyone who has won it.

—
Reply to this email directly, view it on GitHub
<https://github.com/orgs/community/discussions/43514#discussioncomment-7492964>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/A242TKB5BBSQO2OFFJJA5XTYDFWPJAVCNFSM6AAAAAATTMWKQ2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TIOJSHE3DI>
.
You are receiving this because you are subscribed to this thread.Message
ID: ***@***.***>

**Comment by SergeyShurkhovetckii** [comment]
Hi , while this badge is unavailable

**Comment by andriimazurets** [comment]
❤️

**Comment by kirubel606** [comment]
# 💖 Heart On Your Sleeve Badge 💖

Here is how you earn the "Heart On Your Sleeve" badge! 🎉

This badge is a symbol of your empathy, kindness, and genuine care for others within the community. Your willingness to wear your heart on your sleeve and contribute positively has not gone unnoticed.

Thank you for making our community a warm and welcoming place for everyone. Your actions truly embody the spirit of camaraderie and support.

Keep spreading the love and continue being an inspiration to others! 💕

**Reply by HIMELha** [reply]
How can i get it? is it available?

**Comment by Madhulika3081** [comment]
I think they aren't available at the moment.

**Comment by alpersarac** [comment]
Can you react with ❤️  emoji to this response so I can get Heart On Your Sleeve Badge.

**Comment by alisenola** [comment]
Is there anyone who got the Sleeve Badge?
If so, please teach me how to do it.
Crossing my fingers to hear from you at an early date!
Warm regards.

**Comment by ZawMinKhantOrLeon** [comment]
I am little late but please someone <3

**Comment by gchikovanii** [comment]
Nice ))

**Comment by mohmmadView** [comment]
niceee

**Comment by HIMELha** [comment]
Can i get it? is it still available? what is the benifits of badges?

**Reply by Akidul-islam** [reply]
Sponsorship and contributions to the GitHub community are recognized through badges, which enhance your profile and showcase your interests. These badges foster a sense of community and can improve your reputation both within and outside the community.

**Comment by nhanvnguyen** [comment]
help me get it. thanks everyone :D :D

**Comment by alpersarac** [comment]
Can you guys star ⭐  my repositories? I will star yours back. It might work. My friend got it in this way.

https://github.com/alpersarac?tab=repositories

**Reply by HIMELha** [reply]
I'm giving some, don't forgot to give some star on my repo

**Comment by Akidul-islam** [comment]
To acquire the "Heart On Your Sleeve" badge on GitHub, sponsor an open-source developer through GitHub Sponsors. Visit the developer's GitHub profile, click "Sponsor," and follow the steps to become a sponsor. Your support will be reflected on your profile with the badge, showcasing your commitment to open source

**Reply by walidbosso** [reply]
That is a different badge my friend, not the one he's asking about which is still not released, focus my friend focus!! xD 
⎯⎯⎯⍟✪⍟⎯⎯⎯
**But** `some say` **if you put a heart ❤️ on**`my comment specifically` **you may get it**, but who knows ..  ¯\_(ツ)_/¯

**Comment by shenjingnan** [comment]
Can i get it? is it still available? what is the benifits of badges?  ❤️

**Comment by ZubkoKarina** [comment]
❤️

**Comment by ThesanduThathsara** [comment]


**Reply by ljharb** [reply]
This is false.

**Comment by ing9990** [comment]
❤️

**Comment by denys-petryniak** [comment]
Thanks in advance 🙏

**Comment by MaulidanFazri** [comment]
please give me a heart♥️

**Comment by atilagulers** [comment]
I need your heart ❤️

**Comment by mranv** [comment]
can i get that too

**Comment by Navya2ak** [comment]
❤️

**Reply by gmadharh** [reply]
❤️

**Comment by gmadharh** [comment]
hello ❤️

**Reply by Jonak-Adipta-Kalita** [reply]
Hello Bro!! :heart:

**Comment by themagicalmammal** [comment]
This discussion has been the default ❤️ discussion.

**Comment by alexandroivaldez** [comment]
Spreading the ❤️

**Comment by mvuco00** [comment]
hi hi ❤️

**Comment by ettaboyle** [comment]
FYI: Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, **we’ve made the difficult decision to disable the ability to earn Achievements in our Community.**

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by Darrshan-Sankar** [comment]
❤
